app/code/Magento/InstantPurchase/Model/ShippingMethodChoose/DeferredShippingMethodChooserPool.php

```diff
@@ -28,6 +31,10 @@ public function __construct(array $choosers)
         $this->choosers = $choosers;
     }
 
+    /**
+     * @param mixed $type
```

Comment:
`mixed` pseudo-type must be avoided. Please specify exact type(s).

Reply:
Both fixed in 885f032